lightstep/sdk/metric/exporters/otlp/otelcol/client.go

```diff
@@ -144,11 +144,7 @@ func WithTLSSetting(tlss configtls.ClientConfig) Option {
 func NewExporter(ctx context.Context, cfg Config) (metric.PushExporter, error) {
 	c := &client{}
 
-	if !cfg.Exporter.Arrow.Disabled {
```

Comment:
The documentation for the field `cfg.Exporter.Arrow.Disabled` says:

```
        // Disabled prevents using OTel-Arrow streams.  The exporter
        // falls back to standard OTLP.
        Disabled bool `mapstructure:"disabled"`
```

So I think it's safe to just blindly use the otelarrow factory below and let it internally do the fallback as documented.